applications/nfc/nfc.c

```diff
@@ -224,6 +232,21 @@ void nfc_show_loading_popup(void* context, bool show) {
 }
 
 int32_t nfc_app(void* p) {
+    if(!furi_hal_nfc_is_init()) {
```

Comment:
One PR - one feature. Please, checkout from dev

Reply:
Oh, I think I accidentally checked out the wrong branch, yep

Reply:
Fixed!